ext/opentelemetry-ext-system-metrics/tests/test_system_metrics.py

```diff
@@ -0,0 +1,212 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+from unittest import mock
+
+from opentelemetry import metrics
+from opentelemetry.ext.system_metrics import SystemMetrics
+from opentelemetry.sdk.metrics import Observer
+from opentelemetry.test.test_base import TestBase
+
+
+class TestSystemMetrics(TestBase):
+    def test_system_metrics_constructor(self):
+        # ensure the observers have been registered
+        meter = metrics.get_meter(__name__)
+        with mock.patch("opentelemetry.metrics.get_meter") as mock_get_meter:
+            mock_get_meter.return_value = meter
+            SystemMetrics(self.memory_metrics_exporter)
+        self.assertEqual(len(meter.observers), 5)
```

Comment:
Wouldn't be better to check the names of the registered observers instead? This test would pass if the same observer gets registered 5 times .

Reply:
Yup, done